Global displacement

aleesteele opened the pull request:
Basic global displacement chart, no styling

Commit:
global displacement scenarios spline chart: basic without SOURCE*

Commit:
forgot to change axis

Commit:
took out legend title

Commit:
capitol of

Commit:
cleaned up code before pull request

jnschrag reviewed (changes requested):
- Since we're starting from scratch with these, can you go ahead and remove the jQuery dependency?
- Please update the `<title>` tag with the title of the chart
- The zoom in feature is nice to have, however, the X axis can start having decimals on them. Can you look at making sure the year numbers stay whole numbers?

Commit:
kept x-axis whole numbers, changed title, vanilla js

Commit:
full title & deleted jquery label

Commit:
rounded labels for points

Commit:
removed 'milliions' & will ask if needed**

aleesteele commented:
- refactored for vanilla javascript 
- deleted jquery label & removed library
- kept x-axis whole numbers (years)
- rounded point values & removed extra x-axis label
- better refactoring (see most recent commit)

Q's: 
- should I add 'millions' to the point labels themselves? 
- should y-axis labels remain uppercase 'M''s or lowercase? (caroline answered!)

Commit:
better refactoring - other was extraneous

Commit:
first theme commit

Commit:
no 0 value on axis + not ending on tick

Commit:
colors of gridlines + y-axis labels

aleesteele commented:
- Horizontal wide gridlines
- Capitalized y-axis labels / custom labels w/ mil
- Normal weight legend

Q's:
- Didn't change # of x-axis ticks, there might be too much

jnschrag commented on globals/ppd-highcharts-theme.js:
```diff
@@ -0,0 +1,59 @@
+    style: {
+      fontSize: "13px",
+    },
+    headerFormat: "<span style=\"font-size: 13px;text-align:center;margin-bottom: 5px;font-weight: bold;font-family: 'Source Sans Pro', arial, sans-serif;\">{point.key}</span><br/>"
```
Let's remove the reference to "Source Sans Pro" here and make it Proxima Nova like the rest of the chart.

jnschrag reviewed (changes requested):
This is really good, nicely done! Just a few small tweaks and then I'll accept it.

- Can you make sure that in `index.html` all of the external sources have `https:` at the beginning of their URLs?
- Make a copy of the iframeSizer and put it in PPD's global folder, then include it in the chart's `index.html`
- Add "People" as a value suffix in the tooltip

aleesteele committed:
oh, wow

aleesteele committed:
revised after forever

aleesteele commented:
Fixed this one up too!
